src/tests/ftest/osa/osa_offline_reintegration.py

```diff
@@ -24,111 +25,195 @@ def setUp(self):
+                                                               rank[val])
                 self.log.info(output)
-                output = self.dmg_command.system_start(ranks=rank)
-
-            self.log.info(output)
-            self.is_rebuild_done(3)
-            self.assert_on_rebuild_failure()
-
-            pver_exclude = self.get_pool_version()
-            self.log.info("Pool Version after exclude %s", pver_exclude)
-            # Check pool version incremented after pool exclude
-            # pver_exclude should be greater than
-            # pver_begin + 8 targets.
-            self.assertTrue(pver_exclude > (pver_begin + 8),
-                            "Pool Version Error:  After exclude")
-            output = self.dmg_command.pool_reintegrate(self.pool.uuid,
-                                                       rank)
-            self.log.info(output)
-            self.is_rebuild_done(3)
-            self.assert_on_rebuild_failure()
-
-            pver_reint = self.get_pool_version()
-            self.log.info("Pool Version after reintegrate %d", pver_reint)
-            # Check pool version incremented after pool reintegrate
-            self.assertTrue(pver_reint > (pver_exclude + 1),
-                            "Pool Version Error:  After reintegrate")
+                self.is_rebuild_done(3)
+                self.assert_on_rebuild_failure()
 
-        for val in range(0, num_pool):
-            display_string = "Pool{} space at the End".format(val)
-            self.pool = pool[val]
-            self.pool.display_pool_daos_space(display_string)
+                pver_reint = self.get_pool_version()
+                self.log.info("Pool Version after reintegrate %d", pver_reint)
+                # Check pool version incremented after pool reintegrate
+                self.assertTrue(pver_reint > (pver_exclude + 1),
+                                "Pool Version Error:  After reintegrate")
 
-        if data:
-            self.run_ior_thread("Read", self.ior_dfs_oclass[0],
-                                self.ior_apis[0], self.ior_test_sequence[0])
+            display_string = "Pool{} space at the End".format(random_pool)
+            self.pool = pool[random_pool]
+            self.pool.display_pool_daos_space(display_string)
 
-    def test_osa_offline_reintegration(self):
-        """Test ID: DAOS-4749
+        # Finally check whether the written data can be accessed.
+        # Also, run the daos cont check (for object integrity)
+        for val in range(0, num_pool):
+            self.pool = pool[val]
+            if data:
+                self.run_ior_thread("Read", oclass, test_seq)
+                self.run_mdtest_thread()
+                self.container = self.pool_cont_dict[self.pool]
+                #kwargs = {"pool": self.pool.uuid,
+                #          "cont": self.container.uuid}
+                #output = self.daos_command.container_check(**kwargs)
+                #self.log.info(output)
```

Comment:
Is there any reason we don't call daos cont check?

Reply:
daos cont check functionality seems to be broken... Created DAOS-7013 for the issue.

Reply:
Added skipForTicket to avoid commenting the code.